### ISSUE TITLE
Exception in thread "main" java.lang.UnsupportedOperationException: getNewTempClassLoader is not supported

### ISSUE THREAD

**Issue by reylibutan** [scottescue/dropwizard-entitymanager#16]
I follow your tutorial here
http://scottescue.com/dropwizard-entitymanager/

But unfortunately came across this error when I add
`bootstrap.addBundle(entityManagerBundle);`

Also, I added my own

```
        <dependency>
            <groupId>org.hibernate</groupId>
            <artifactId>hibernate-entitymanager</artifactId>
            <version>5.1.0.Final</version>
        </dependency>
```

Since the one in your lib is outdated.

**Comment by scottescue** [comment]
I think you forgot to show the error you're getting.

**Comment by reylibutan** [comment]
I don't know but when I start the project, the exception (in the issue title) is the only one being shown.

**Comment by scottescue** [comment]
Apologies, I was on mobile and didn't notice the exception in the title.  I won't have time to try to completely duplicate this until later tonight.  But, I believe this is going to be caused by your usage of Hibernate 5.  

I did some quick debugging during my break; it appears the Hibernate team reworked how the PersistenceUnitInfo is processed beginning with Hibernate 5.  Their changes now include calls to the PersistenceUnitInfo#getNewTempClassloader method.  Hibernate 4 didn't make those calls, so the implementation of PersistenceUnitInfo in DW EM didn't support the method.

The 0.9.x versions of DW EM were only intended to work with Hibernate 4, so they haven't been tested with Hibernate 5.  I'll be working on the 1.0.0-1 release over the next week or so. 1.0.0-1 will include an upgrade to Hibernate version 5.1.0.Final.  I've been purposefully matching the Hibernate versions used in corresponding releases of Dropwizard Hibernate, which didn't upgrade to Hibernate 5 until the 1.0.0 release.

**Comment by reylibutan** [comment]
No worries and thank you for the great plugin!

I'll try work with Hibernate 4 at the mean time and change back when a new release is available.

Great work man!

**Comment by scottescue** [comment]
@reylibutan I've got Hibernate 5 successfully working, but it's going to be at least a few days before I have a release ready.  I pushed a snapshot build to Sonatype OSS, if you care to take a look.

``` xml
    <repositories>
        <repository>
            <id>oss-sonatype-snapshots</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
            <snapshots>
                <enabled>true</enabled>
                <updatePolicy>always</updatePolicy>
            </snapshots>
        </repository>
    </repositories>

    <dependencies>
        ...
        <dependency>
            <groupId>com.scottescue</groupId>
            <artifactId>dropwizard-entitymanager</artifactId>
            <version>1.0.0-1-SNAPSHOT</version>
        </dependency>
        ...
    </dependencies>
```

**Comment by scottescue** [comment]
This is resolved with the 1.0.0-1 release
